README.md

```diff
@@ -0,0 +1,9 @@
+# Matcha-dark-azul on Flathub
```

Comment:
This is not really needed. Setup instructions are already there on our website and `flatpak update` will install matching theme automatically.